src/Components/CatProfile.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { useState, useEffect, ChangeEvent } from 'react';
+import { getCatsProfiles } from '../api';
+
+function CatProfile() {
+	const [changeName, setChangeName] = useState<string>('');
+	const [changePictureurl, setChangePictureurl] = useState<string>('');
+	const [changeDescription, setChangeDescription] = useState<string>('');
+	const [catsProfile, setCatsProfile] = useState<any>(''); //it will be an interface
+	const [isLoading, setIsLoading] = useState<boolean>(true);
+
+	useEffect(() => {
+		getCatsProfiles().then((fetchedProfiles) => {
+			if (Array.isArray(fetchedProfiles) && fetchedProfiles.length > 0) {
+				const profile = fetchedProfiles[0];
+
+				console.log(fetchedProfiles);
+				setCatsProfile(fetchedProfiles);
+				setIsLoading(false);
+				setChangeName(profile.name);
+				setChangePictureurl(profile.pictureurl);
+				setChangeDescription(profile.description);
+			}
+		});
+	}, []);
+
+	if (isLoading) {
+		return <p>Loading...</p>;
+	}
+
+	const handleProfileEditBy = (event: ChangeEvent<HTMLInputElement>) => {
+		//Handle editing process of Cat's profile, allow users to edit the profile
+		const { name, value } = event.target;
+
+		if (name === 'name') {
+			setChangeName(value);
+		} else if (name === 'pictureurl') {
+			setChangePictureurl(value);
+		} else if (name === 'description') {
+			setChangeDescription(value);
+		}
+	};
+
+	const handleSavedChanges = () => {
+		console.log({ changeName, changePictureurl, changeDescription });
+	};
+
+	return (
+		<div className="profile-container">
+			<h2 className="profile-title">Cat's Profile</h2>
```

Comment:
This produces a linting error - always run the code and check for errors before starting a PR.

> [eslint] 
src/Components/CatProfile.tsx
  Line 50:37:  `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`  react/no-unescaped-entities

(It means you need to replace ' with &apos; because it's in HTML.